DMTN-107.tex

```diff
@@ -178,28 +180,36 @@ \subsection{Do Nothing}\label{ssec:potsol_donothing}
 \end{tabular}
 \end{center}
 
-\textcolor{blue}{ Scope -- No expansion of DM scope.}
-This option should be considered a de-scope (and would likely be perceived as a de-scope by the community).
+\textcolor{blue}{ Scope -- No expansion of DM scope.} 
+Although not strictly speaking a de-scope, this option is likely be perceived as a de-scope by the community and will fall below their expectations. 
+
+As an additional note, LSST Education and Public Outreach (EPO) are not expecting to serve alerts to the community prior to Data Release 1 and the beginning of LSST Alert Stream production \citedsp{LEP-031}, and so this option would not have any scope implications for EPO. 
 
 \textcolor{orange}{ Risks -- Potential risks for the DMS.}
-There is a risk to DM in missing the opportunity to learn lessons from running alert production in LOY1.
-This puts a higher importance on running alert production during commissioning, and/or imposes a higher risk on alert production when it finally does start in year 2.
-There is another risk to the DMS: it is likely that the community will do DIA and/or alert production with user-generated pipelines based on the processed standard-visit images released within 24 hours\reqparam{L1PublicT}\lsrreq{0104}.
+There is a risk to DM in missing the opportunity to learn lessons from running alert production in LOY1
+% LPG: I don't think the decision to not run AP in LOY1 will change anything during commissioning If anything, 
+% there will be less of a reason to produce templates during commissioning as they won't be used in LOY1. If commissioning gets squished, this will be an easier task to drop 
+% This puts a higher importance on running alert production during commissioning  
```

Comment:
start of the sentence is commented out here?

Reply:
No - this was the part of the sentence I propose to remove.